PlayerUI/Views/PUIButtonView.swift

```diff
@@ -57,18 +57,18 @@ import SwiftUI
 /// **Future Direction**: This will eventually be replaced by a pure SwiftUI implementation
 /// once all complex AppKit dependencies are resolved.
 public struct PUIButtonView: View {
-    @State private var button: PUIButton
+    @StateObject private var button: PUIButton
```

Comment:
This is what I meant to do to begin with. The before state was leaking PUIButton's.